recipes/libmodplug/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "0.8.9.0":
+    url: "https://github.com/Konstanty/libmodplug/archive/5a39f5913d07ba3e61d8d5afdba00b70165da81d.zip"
```

Comment:
I don't think this is the official 0.8.9.0 release but a modified one.
The project is on sourceforge.
Shouldn't it be https://netcologne.dl.sourceforge.net/project/modplug-xmms/libmodplug/0.8.9.0/libmodplug-0.8.9.0.tar.gz ?

Reply:
When you go to their website http://modplug-xmms.sourceforge.net you see there a link to this GitHub repository. It is official

Reply:
Thanks for the ref.
Too bad there is no *official* release with the cmake script.
Why did you choose this commit? Shouldn't the name of the version be the date of that commit instead of `0.8.9.0`?

Reply:
We only do date versioning when there no official releases, but they do official releases on SourceForge.

The commit should be equal to what is the latest version of libmodplug on SourceForge. When you look at the tree you see that the latest commit changing the libmodplug directory is `[f97ba4] Bump libmodplug version to 0.8.9.0` https://sourceforge.net/p/modplug-xmms/git/ci/master/tree/

If there are concerns about this I can switch the download to SourceForge, but since the repository is official I would prefer GitHub over SourceForge

Reply:
My only issue with this is that the commit hash does not feel like a release.
Also, I don't think the sourceforge repo contains a cmake script.

Reply:
The SourceForge repo contains the CMakeLists.txt file https://sourceforge.net/p/modplug-xmms/git/ci/master/tree/libmodplug/

The SourceForge download tarball doesn't

In the Bincrafters version we had download for a long time the SourceForge tarball and the CMakeLists.txt file separately https://github.com/bincrafters/conan-libmodplug/commit/4ebc9c073ac131bc39eb4427bf3211d0fab125ce

But I think that this is nonsense

We could also download a git repository tarball from SourceForge https://sourceforge.net/p/modplug-xmms/git/ci/master/tarball?path=/libmodplug

Reply:
Vcpkg is download from GitHub too https://github.com/microsoft/vcpkg/blob/1be75a28bfce2084bb0b59ec92a6853f13d7be66/ports/libmodplug/portfile.cmake#L5

Reply:
I agree that it is confusing so I have created https://github.com/Konstanty/libmodplug/issues/48

Reply:
Thanks for the clarification.
I just read your issue in the libmodplug repo.
Letting upstream decide for us is always easier :smile: